### ISSUE TITLE
Questions about your early stop strategy during reproducting your results

### ISSUE THREAD

**Issue by my462** [SNAS-Series/SNAS-Series#5]
Hi, I am trying to reproduce your experment result and I just run your step one with both commends:
python -m torch.distributed.launch --nproc_per_node=8 train_imagenet.py \
--SinglePath --bn_affine --flops_loss --flops_loss_coef 1e-6 --seed 48 --use_dropout --gen_max_child --early_fix_arch --config configs/SinglePath240epoch_arch_lr_1e-3_decay_0.yaml \
--remark 'search_arch_lr_1e-3_decay_0'
and the one with 30 pretrain epochs:
python -m torch.distributed.launch --nproc_per_node=8 train_imagenet.py \
--SinglePath --bn_affine --flops_loss --flops_loss_coef 1e-6 --seed 48 --use_dropout --pretrain_epoch 30 --gen_max_child --early_fix_arch --config configs/SinglePath240epoch_arch_lr_1e-3_decay_0.yaml \
--remark 'search_arch_lr_1e-3_decay_0'
Both experiments now have complete around 160 epochs, but I can only see 8 of 20 layers structures have met the early stop condition. It's not like what you said in another issue that around training after 80 epochs, almost all the structures are fixed according to the early stop stragety.
Can you help me with this?
Thx~

**Comment by skhu101** [comment]
Thanks for your interest in our paper! 

According to our experiments, I think the relative order of alpha should not change so much even if some layers may not meet the condition. Anyway, could you send us the updating curve of architecture parameters alpha? We will have a check. Email: skhu [at] se [dot] cuhk [dot] edu [dot] hk

**Comment by my462** [comment]
> Thanks for your interest in our paper!
> 
> According to our experiments, I think the relative order of alpha should not change so much even if some layers may not meet the condition. Anyway, could you send us the updating curve of architecture parameters alpha? We will have a check. Email: skhu [at] se [dot] cuhk [dot] edu [dot] hk
Sorry for replying this late.
Just check the log alpha result in 80th and 240th and the selected block results are as follows,
80th   [0, _**3**_, 1, 0, 3, 1, 1, 0, 3, _**3**_, _**3**_, 3, 3, 3, 0, 0, 0, _**3**_, _**3**_, 0]  5 layers are early stopped
120th [2, _**3**_, 0, 1, 0, 0, 2, _**3**_, _**3**_, _**3**_, _**3**_, 3, 0, 0, 0, _**3**_, _**3**_, _**3**_, _**3**_, _**0**_]  10 layers are early stopped
Our lab machine is offline so I can't send you the email about more detailed result.
